packages/admin-ui/src/table/resolvers/text.tsx

```diff
@@ -47,6 +65,7 @@ export function textResolver<T>() {
 export type TextResolver<T> = {
   type: 'text'
   columnType?: 'name' | 'text'
+  ellipsis?: boolean
```

Comment:
What do you think of something like that? Instead of being a boolean, IMO this can more scalable if we want to add another option of overflow in the future
```suggestion
  overflow?: 'ellipsis' | 'auto'
```

Reply:
good suggestion ;)